test/pegin-address-verifier.js

```diff
@@ -128,3 +128,46 @@ describe('decodeWifPrivateKey tests', () => {
         expect(utils.decodeWifPrivateKey(wif)).to.be.eq('ef235aacf90d9f4aadd8c92e4b2562e1d9eb97f0df9ba3b508258739cb013db2');
     });
 });
+
+describe('createPeginV1TxData tests', () => {
+
+    it('should create pegin v1 data without btc refund address', () => {
+        const rskAddress = '0xa65C9bF91Ae59fC5bE36300D0BC271998529bb75';
+        const expectedData = '52534b5401a65C9bF91Ae59fC5bE36300D0BC271998529bb75';
+        expect(validator.createPeginV1TxData(rskAddress)).to.be.eq(expectedData);
+    });
+
+    it('should create pegin v1 data with rsk address without the 0x prefix', () => {
+        const rskAddress = 'a65C9bF91Ae59fC5bE36300D0BC271998529bb75';
+        const expectedData = '52534b5401a65C9bF91Ae59fC5bE36300D0BC271998529bb75';
```

Comment:
should the expected data be in lower case or it doesn't make a difference?

Reply:
It doesn't make a difference. It's hex data. But as a standard now I'm returning it in lowercase.